lib/reek/smells/uncommunicative_method_name.rb

```diff
@@ -49,17 +50,17 @@ def inspect(context)
 
       private
 
-      def acceptable_name?(name:, context:)
-        accept_names(context).any? { |accept_name| name == accept_name } ||
-          reject_patterns(context).none? { |pattern| name.match pattern }
+      def acceptable_pattern?(name:, context:)
```

Comment:
I would keep the name `acceptable_name?`, since that's what you're checking.

`accept_patterns` etc. are all good name-wise.